### DISCUSSION TITLE
How can I do the same in Twind?

### DISCUSSION THREAD

**Discussion by babakfp** [tw-in-js/twind#391] [Q&A]
https://tailwindcss.com/docs/plugins#adding-base-styles

**Comment by sastan** [comment]
Which version are you using v0.16 or v1.0?

**Reply by babakfp** [reply]
0.16.17

**Reply by sastan** [reply]
Sorry for the late answer. This should be possible with the `preflight` option: https://twind.dev/handbook/configuration.html#preflight

**Reply by gera2ld** [reply]
Hi, is there an equivalent for this in v1.0?

**Reply by sastan** [reply]
here is an example using the `preflight` config option: https://twind.run/audio-solar-bridge?file=config&result=css

```js
import { defineConfig, css } from "@twind/core"
import presetAutoprefix from "@twind/preset-autoprefix"
import presetTailwind from "@twind/preset-tailwind"

export default defineConfig({
  presets: [
    presetAutoprefix(),
    presetTailwind(/* options */),
  ],
  preflight: css`
    input[type='number'],
    input[type='text'] {
      -moz-appearance: textfield;
      appearance: textfield;
      outline: none;
    }
    input::-webkit-outer-spin-button,
    input::-webkit-inner-spin-button {
      -webkit-appearance: none;
      outline: none;
    }
  `
})
```

Source: https://discord.com/channels/798324011980423188/938703212469440542/1064875861108609084

To access the theme you can use the [theme helper](https://tailwindcss.com/docs/functions-and-directives#theme).
